packages/image_picker/lib/image_picker.dart

```diff
@@ -79,16 +91,31 @@ class ImagePicker {
   /// The [source] argument controls where the video comes from. This can
   /// be either [ImageSource.camera] or [ImageSource.gallery].
   ///
+  /// The [quality] optional argument controls the video quality and its size on correspondence. This can
```

Comment:
@cyanglaz should we enforce 80 char line length?

Reply:
I break it into two Lines, as the default value was originally in a separate statement.